Backend/Controllers/SiteRoutes/PageController.cs

```diff
@@ -79,7 +79,7 @@ public async Task<IActionResult> NewPage([FromRoute] string shopId, [FromBody] N
     }
 
     public record UpdatePageInput(string? Slug, string? Title, string? Description, string? Content);
-
+    public record PageOutput(string Slug, string? Title, string? Content, string? Description);
```

Comment:
Why `PageOutput` have all fields nullable if they are not nullable? 
![image](https://github.com/flurium/spentoday-api/assets/103932583/01b20e61-d931-4716-8a3d-49dbcd9f2ca0)

Reply:
That's how it was. Strings, not nullable strings